govuk-social-providers/login/template.ftl

```diff
@@ -239,22 +239,12 @@
 
     <div id="global-app-error" class="app-error hidden"></div>
 
-    <script src="${url.resourcesPath}/vendor/govuk_template/javascripts/govuk-template.js?0.19.2"></script>
+    <script src="${url.resourcesPath}/vendor/govuk_template/javascripts/govuk-template.js?0.22.1"></script>
 
     <script>if (typeof window.GOVUK === 'undefined') document.body.className = document.body.className.replace('js-enabled', '');</script>
 
     <#-- End GOV.UK Template footer -->
-
-
-    <#-- Enable GOV.UK Front-End Toolkit checkboxes -->
-    <script src="${url.resourcesPath}/vendor/jquery/jquery.min.js"></script><#-- GOV.UK Front-End Toolkit selection buttons (radios and checkboxes) currently depend on jQuery -->
-    <script src="${url.resourcesPath}/vendor/govuk_frontend_toolkit/javascripts/govuk/selection-buttons.js"></script>
     <script src="${url.resourcesPath}/javascripts/details.polyfill.js"></script>
-    <script>document.addEventListener('DOMContentLoaded', function (event) {
-        new GOVUK.SelectionButtons($('label input[type=checkbox]'));
-    });
-    </script>
```

Comment:
Yup — as of (I think 3.0) they’re CSS-only.

The CSS in question doesn’t work in IE 8, so you just get plain, regular-sized radios and checkboxes there. I think we’ll try to polyfill in the modern browser appearance in IE 8 on our main project, but we’re happy to leave the default in the Keycloak theme for now (as we don’t use any of Keycloak’s checkbox options).